compose/aspnet-mssql-compose.md

```diff
@@ -0,0 +1,142 @@
+---
+description: Create a Docker Compose application using ASP.NET Core and SQL Server on Linux in Docker. 
+keywords: dotnet, .NET, Core, example, ASP.NET Core, SQL Server, mssql
+title: "Quickstart: Compose and ASP.NET Core with SQL Server"
+---
+
+This quick-start guide demonstrates how to use Docker Compose to set up and run the sample ASP.NET Core application using the [ASP.NET Core Build image](https://hub.docker.com/r/microsoft/aspnetcore-build/) with the [SQL Server on Linux image](https://hub.docker.com/r/microsoft/mssql-server-linux/). You just need to have [Docker Engine](https://docs.docker.com/engine/installation/) and [Docker Compose](https://docs.docker.com/compose/install/) installed on your platform of choice.
+
+For this sample, we will create a sample `dotnet core web application` using the `aspnetcore-build` Docker image. After that, we will create a `Dockerfile`, configure this app to use our SQL Server database and then create a `docker-compose.yml` that will define the behavior of all of these components.
+
+# Create the Docker Compose application
+
+1. Create an empty directory named `/app` under the root and navigate into it.
+
+    This directory will be the context of your docker-compose project. For [Docker for Windows](https://docs.docker.com/docker-for-windows/#/shared-drives) and [Docker for Mac](https://docs.docker.com/docker-for-mac/#/file-sharing), you have to set up file sharing for the volume that you need to map. 
+
+1. Within your directory, use the `aspnetcore-build` Docker image to generate a sample web application within the container and into your host machine under `/app`:
+
+    ```bash
+    docker run -v /app:/app microsoft/aspnetcore-build:lts /bin/bash -c "cd app && dotnet new -t web"
+    ``` 
+
+    >**Note**: If running in Docker for Windows, make sure to add the necessary `C:/` prefix to the host mapped folder.
```

Comment:
Addressed with this commit: https://github.com/docker/docker.github.io/pull/1536/commits/62efa562018a786afa2a9261910b6ecebb21a167